packages/pagination/src/PaginationContent.js

```diff
@@ -90,6 +94,22 @@ const PaginationContent = ({
     );
   }
 
+  const items =
+    page &&
```

Comment:
You're going to want to create a memoized version of this in case there are unnecessary re-renders happening.

Reply:
c97eca6842d986fc5021400eec82b82386496802